docs/source/developer/extension_dev.rst

```diff
@@ -191,9 +191,8 @@ subsequently reversed by running
 
     jlpm run remove:package <extension-dir-name>
 
-This will remove the package metadata from the source tree, but will
-**not** remove any files added by the ``addsibling`` script, which
-should be removed manually.
+This will remove the package metadata from the source tree and remove any
+files added by the ``addsibling`` script.
```

Comment:
I think it will also remove files for packages that were *not* added by the `add:sibling` script. Perhaps we can just say "delete all of the package files"?

Reply:
Alright I think I've fixed it! Thanks for the feedback.